src/Docker.DotNet/Models/TLSInfo.Generated.cs

```diff
@@ -10,9 +9,9 @@ public class TLSInfo // (swarm.TLSInfo)
         public string TrustRoot { get; set; }
 
         [DataMember(Name = "CertIssuerSubject", EmitDefaultValue = false)]
-        public IList<byte> CertIssuerSubject { get; set; }
+        public string CertIssuerSubject { get; set; }
```

Comment:
Hmm? Are you sure you can pass a cert this way? Seems wrong

Reply:
Tests fail when setting this to IList<byte> and haven't found any workarround

Reply:
Ok np. Does appear to be a string in the json object

Reply:
It was done in the go code generator.

Reply:
It was updated via the go code generator. My intention is to provide the minimum set of changes for tests to work, and continue adding improvements and collaborations from there.

@The-TT-Hacker I agree it would be a better solution but a minimum working base is required for any contribution to pass the CI/CD tests, get reviewed, and go ahead.

Reply:
I dont think it's a good idea to be overriding the model types and then having to make a breaking change to the client library to revert to the correct solution. The proper fix is very minimal and not hard to include.

Reply:
Done